bin/buildpack_start.sh

```
@@ -0,0 +1,18 @@
+
+# Start the Django backend server
+uvicorn --app-dir=/app --host=0.0.0.0 --timeout-graceful-shutdown=300 --limit-max-requests=20000 --lifespan=off impress.asgi:application &
+
+# Start the Y provider service
+cd src/frontend/servers/y-provider && PORT=4444 ${NODE_BIN:-node} dist/start-server.js &
+
+# Start the Nginx server
+bin/run &
+
+# if the current shell is killed, also terminate all its children
+trap "pkill SIGTERM -P $$" SIGTERM
```

Shellcheck (0.11.0)

[warning] Use single quotes, otherwise this expands now rather than when signalled.

(SC2064)